langserver/handler_common.go

```diff
@@ -27,10 +26,10 @@ func (h *HandlerCommon) Reset(rootURI string) error {
 	if h.shutdown {
 		return errors.New("unable to reset a server that is shutting down")
 	}
-	if !strings.HasPrefix(rootURI, "file:///") {
+	if !isUri(rootURI) {
```

Comment:
in some places we trick triple forward slash, in other only two slashes. However, you have replaced all implementations with just `isUri` which checks triple forward slash.

Is the places we checked double forward slash just not as correct and this change is correct?